modules/virt-resuming-node-from-maintenance-mode-with-cr.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// virt/node_maintenance/virt-resuming-node.adoc
+
+[id="virt-resuming-node-from-maintenance-mode-with-cr_{context}"]
+= Resuming a node from maintenance mode that was initiated with a `NodeMaintenance` CR
```

Comment:
with a NodeMaintenance CR

(we are doing away with markdown in titles)

Reply:
oops! Fixed this now.